benefits/enrollment_switchio/api.py

```diff
@@ -0,0 +1,126 @@
+from dataclasses import dataclass
+from datetime import datetime
+from enum import Enum
+import hashlib
+import hmac
+import json
+import requests
+
+
+@dataclass
+class Registration:
+    regId: str
+    gtwUrl: str
+
+
+class EshopResponseMode(Enum):
+    FRAGMENT = "fragment"
+    QUERY = "query"
+    FORM_POST = "form_post"
+    POST_MESSAGE = "post_message"
+
+
+@dataclass
+class RegistrationStatus:
+    regState: str
+    created: datetime
+    mode: str
+    tokens: list[dict]
+    eshopResponseMode: str
+    identifier_type: str = None
+    mask_cin: str = None
+    card_exp: str = None
+
+
+class Client:
+
+    def __init__(
+        self,
+        api_url,
+        api_key,
+        api_secret,
+        private_key,
+        client_certificate_file,
+        ca_certificate,
+    ):
+        self.api_url = api_url
+        self.api_key = api_key
+        self.api_secret = api_secret
+        self.private_key = private_key
+        self.client_certificate_file = client_certificate_file
```

Comment:
Ugh, I never settled on whether I like `_file` here or not... These parameters, `private_key`, `client_certificate`(`_file`), and `ca_certificate`, are expecting file paths.

I'm just gonna remove `_file` from this

Reply:
I like consistency more than anything :grin: